### PR TITLE
bug to fix validation of fields

### PR DESCRIPTION
**Pull request by bryyoz**
**What this PR does**

- It validates for empty fields

### CODE REVIEW

File: app/api/V1/views/product.py [hunk 1]
```diff
@@ -17,32 +17,36 @@
...
     @ns_product.expect(product_models)
     def post(self):
-      
-
-
-
       args = parser.parse_args()
-      product_name = args['product_name']
+      product_names = args['product_name']
+      for product_name in product_names:
```
Comment (Bonifase) [inline app/api/V1/views/product.py]: Is product_names a list? If it is not, then you need to get rid of the for statement

Comment (bryyoz) [reply app/api/V1/views/product.py]: i have tried that too by  having  if not product_name:return{'cannot be empty"} but i get an error message Object of type set is not JSON serializable

Comment (Bonifase) [reply app/api/V1/views/product.py]: what does this product_names contain?

Comment (bryyoz) [reply app/api/V1/views/product.py]: its suppose to be a string that allows you to input a name of a product during post

Comment (Bonifase) [reply app/api/V1/views/product.py]: just below the product_names, include `print(product_names)`, run the app with post request and see what is printed on the terminal

Comment (bryyoz) [reply app/api/V1/views/product.py]: it returns an empty field

Comment (bryyoz) [reply app/api/V1/views/product.py]: the research i have done while using restplus  when i initialiaze the field to be true it shouldnt allow a user to make a post without filling in the field
capture

Comment (Bonifase) [reply app/api/V1/views/product.py]: use if statement not for

Comment (Bonifase) [reply app/api/V1/views/product.py]: eg. `if not product_names:`

Comment (bryyoz) [reply app/api/V1/views/product.py]: thanks that worked

File: app/api/V1/views/product.py [hunk 2]
```diff
@@ -11,36 +11,50 @@
...
   })
 
 parser = reqparse.RequestParser()
 
-parser.add_argument('product_name')
-parser.add_argument('category')
-parser.add_argument('description')
-parser.add_argument('inventory')
-parser.add_argument('price')
+parser.add_argument('product_name', required=True, help= 'This field cannot be blank')
```
Comment (Bonifase) [inline app/api/V1/views/product.py]: at what point are you using this?

Comment (Bonifase) [reply app/api/V1/views/product.py]: fix failing tests

**Pull request merged**